recipes/upsetplot/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "UpSetPlot" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ad40e6c5bc614bb38dff10d1c4a12ab3e20ea2f978ada8d67ba5ad3bbf195f2
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - pandas
+    - matplotlib >=2.0
+  run:
+    - python
+    - pandas
+    - matplotlib >=2.0
+
+test:
+  imports:
+    - upsetplot
+
+about:
+  home: https://github.com/jnothman/UpSetPlot
+  license: New BSD
+  license_family: BSD
+  summary: 'Creating upsetplots in Python'
+  license_file: LICENSE
```

Comment:
this is out of order, please move below license_family and above summary

Reply:
Fixed!